base_delivery_carrier_label/models/carrier_account.py

```diff
@@ -17,6 +17,10 @@ def _selection_file_format(self):
                 ('XML', 'XML')]
 
     name = fields.Char(required=True)
+    delivery_type = fields.Selection(
```

Comment:
@florian-dacosta Maybe this one should come with a related if you have a 'carrier_id' field ?